js/engine/Makefile.include

```diff
@@ -1,7 +1,8 @@
 EMCC_DEFAULTS = \
    -sALLOW_MEMORY_GROWTH=1 \
    -sEXPORTED_RUNTIME_METHODS=UTF8ToString,stringToUTF8,stringToAscii,lengthBytesUTF8,getValue,setValue,intArrayFromString,writeArrayToMemory \
-   -sMODULARIZE
+   -sMODULARIZE \
+   -sSINGLE_FILE=1
```

Comment:
I have flip-flopped on this a few times in the past -- what's your reasoning for embedding?

Reply:
When in node the wasm loading doesn't work, I'm not 100% sure why but I think it's something to do with how the vscode extensions get packed. Also this is a lot easier to work with imo